src/orb.yml.hbs

```diff
@@ -957,6 +958,7 @@ commands:
               if [ "<< parameters.force-new-deployment >>" == "true" ]; then
                   set -- "$@" --force-new-deployment
               fi
+              echo "Before update-service \$@ is: $@"
```

Comment:
I would like the debug output here to be toggleable as well (see my previous comment)